app/lib/config/config.go

```diff
@@ -237,3 +240,13 @@ func (s *ServerToml) String() string {
 	}
 	return buff.String()
 }
+
+// TildeToHome takes a path and replaces an eventual "~" with the home-directory
+func TildeToHome(path string) string {
```

Comment:
Please try out what you're proposing: `something/~/somethingelse` doesn't use the home-directory.
What is the difference between the solution written there and mine?